src/templates/en/2019/contributors.html

```diff
@@ -302,9 +302,23 @@ <h1 class="title title-lg">
     <div class="contributors-count"><span id="filtered-contributors">{{ config.contributors.items() | length }}</span> contributors</div>
     <div class="filter-container">
       <p>Filter by team:</p>
-    {% for id, team in config.teams.items() %}
-      <button class="{{ id }}" type="button">{{ team.name }}</button>
-    {% endfor %}
+      {% set localizedTeamNames = {
+        "analysts": "Analysts",
+        "authors": "Authors",
+        "brainstormers": "Brainstormers",
+        "designers": "Designers",
+        "developers": "Developers",
+        "editors": "Editors",
+        "reviewers": "Reviewers",
+        "translators": "Translators", 
+        } %}
+        {% for id, team in config.teams.items() %}
+          {% if localizedTeamNames[id]|length %}
```

Comment:
Does the null coalescing `or` not work in this situation (`localizedTeamNames[id] or team.name`)? If it doesn't perhaps set a variable once (inside the `if else`) instead of duplicating the markup for the button.